03_url_shortener/main.go

```diff
@@ -3,14 +3,22 @@ package main
 import (
 	"fmt"
 	"github.com/gin-gonic/gin"
+	"net/http"
 	"url_shortener/handler"
 	"url_shortener/store"
 )
 
 func main() {
 	r := gin.Default()
+
+	// Store initialization
+	storage := store.InitializeStore(store.Config{})
```

Comment:
Since you are passing empty config, your connection can't be established, right?